internal/resources/webform/webform.js

```diff
@@ -2424,6 +2425,19 @@ window.initGRPCForm = function(services, invokeURI, metadataURI, debug, headers)
         if (json) {
             history = (JSON.parse(json))
         }
+
+        $.ajax({
+            url: 's/history.json',
+            type: 'GET',
+            success: function(data){
+                preloadedHistory = data;
+                updateHistoryUI();
+            },
+            error: function(data) {
+                // nothing to do probably no history saved server side
```

Comment:
I think we should actually have a stub in the server that can return an empty JSON document, even if the user has not registered a history file. In fact, to make this less subtle and more discoverable in the Go API, I think it best to add first class support for it to the `standalone` package, in the form of a new `HandlerOption`. That way, you also don't need the "/s/" prefix.

Reply:
I've added `standalone.WithExamples` for this role.